api/src/main/java/org/pmiops/workbench/opsgenie/OpsGenieService.java

```diff
@@ -0,0 +1,9 @@
+package org.pmiops.workbench.opsgenie;
+
+import org.pmiops.workbench.model.EgressEvent;
+
+public interface OpsGenieService {
+
+  // Creates an Opsgenie alert for a high-egress event detected in the Workbench system.
+  public void createEgressEventAlert(EgressEvent egressEvent);
```

Comment:
I'm treating the OpsGenie client as the lower-level library that we're trusting is well-tested enough to use for our higher-level needs (and which is mocked out in unit tests).

And since we're talking about ~dozens of lines of code here, I'm not too worried about not having the perfect / optimal structure. This can be easily changed as our needs shift over time.